src/datepicker/__tests__/datepicker-int.scenario.js

```diff
@@ -9,18 +9,14 @@ LICENSE file in the root directory of this source tree.
 import * as React from 'react';
 
 import {StatefulDatepicker} from '../index.js';
-import {hu} from 'date-fns/locale/index.js';
```

Comment:
Could you keep the locale usage in here?

Reply:
Done